Demo/VIper-v2-Demo/Viper-v2-Demo/Application/AppDelegate.swift

```diff
@@ -22,7 +22,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
         initializers.forEach { $0.performInitialization() }
         
-        self.window?.rootViewController = LoginWireframe().viewController
+        let initialController = UINavigationController()
+        initialController.setRootWireframe(LoginWireframe())
+        
+        self.window?.rootViewController = initialController
```

Comment:
It seems that your window is already created. That can happen when you setup main storyboard in project -> general.
Here it can be a bad thing since you are creating a viewController from your main storyboard and then releasing it without using it.
You can avoid this by removing main view controller from project -> general

Reply:
What Vlaho said but I would just point one stuff, if you remove main view controller from project settings, then it will not create UIWindow and you'll have to create it manually here.